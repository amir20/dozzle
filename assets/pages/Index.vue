<template>
  <div class="hero is-fullheight is-dark">
    <div class="hero-body">
      <div class="container has-text-centered">
        <h1 class="title">Please choose a container from the list to view the logs</h1>
      </div>
    </div>
  </div>
</template>

<script>
export default {
  props: [],
  name: "Default",
};
</script>
<style scoped>
.hero.is-dark {
  color: #ddd;
  background-color: #111;

  html.has-light-theme & {
    background-color: #F8F8F8;

    .title {
      color: #555;
    }
  }
}
</style>
