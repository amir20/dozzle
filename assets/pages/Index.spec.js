import { shallowMount } from "@vue/test-utils";
import Index from "./Index";

describe("<Index />", () => {
  test("renders correctly", () => {
    const wrapper = shallowMount(Index);
    expect(wrapper.element).toMatchSnapshot();
  });
});
