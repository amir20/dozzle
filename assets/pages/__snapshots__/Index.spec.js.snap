// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Index /> renders correctly 1`] = `
<div
  class="hero is-fullheight"
>
  <div
    class="hero-body"
  >
    <div
      class="container has-text-centered"
    >
      <h1
        class="title"
      >
        Please choose a container from the list to view the logs
      </h1>
    </div>
  </div>
</div>
`;
